espnet/bin/st_train.py

```diff
@@ -85,6 +85,10 @@ def get_parser(parser=None, required=True):
     parser.add_argument('--mt-weight', default=0.0, type=float,
                         help='Multitask learning coefficient for MT task, weight: \
                                 mt_weight*mt_loss + (1-mt_weight-asr_weight)*st_loss')
+    parser.add_argument('--lsm-type', const='', default='', type=str, nargs='?', choices=['', 'unigram'],
+                        help='Apply label smoothing with a specified distribution type')
+    parser.add_argument('--lsm-weight', default=0.0, type=float,
+                        help='Label smoothing weight')
```

Comment:
I think this is Transformer-only config and `lsm-type` does not work. If you have seen `asr_train.py`, it is kinda workaround for some historical reason. Please ignore that. 

Can you put this at `add_arguments` in the model definition? https://github.com/espnet/espnet/blob/3a4b65e95289fabbf6500d75336a60085e61c42e/espnet/nets/pytorch_backend/e2e_st_transformer.py#L42

Reply:
This is for RNN-based models.